.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: svp-harvester CI
+
+on:
+  pull_request:
+    branches: [ "dev-main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: svp_harvester_test
+          POSTGRES_PASSWORD: svp_harvester_test
+          POSTGRES_DB: svp_harvester_test
```

Comment:
this will run the tests with the postgres root user: couldn't this hide a permissions bug?